apps/broker-service/.vscode/launch.json

```diff
@@ -17,10 +17,7 @@
         "ts-node/register",
         "${workspaceFolder}/src/main.ts",
       ],
-      "envFile": "${workspaceFolder}/../../environments/debug.env",
-      "env": {
-        "SETTINGS": "live"
-      },
+      "envFile": "${workspaceFolder}/../../environments/devlive.env",
```

Comment:
This project cannot function without a database, and it turns out our mock doesn't run without jest :-(

Maybe what I should do is load jest on startup?

Reply:
running this without devlive would be way easier...

Reply:
https://github.com/thecointech/thecoin/issues/184